### PR TITLE
[pl] update date to mention KubeCon NA 2022

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/30722/files
/assign @mfilocha

### CODE REVIEW

File: content/pl/_index.html [hunk 1]
```diff
@@ -44,12 +44,12 @@ <h2>The Challenges of Migrating 150+ Microservices to Kubernetes</h2>
...
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna21" button id="desktopKCButton">Weź udział w KubeCon North America 11-15.10.2021</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2022/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu22" button id="desktopKCButton">Weź udział w KubeCon Europe 17-20.06.2022</a>
         <br>
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2022/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu22" button id="desktopKCButton">Weź udział w wirtualnym KubeCon Europe 17-20.5.2022</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna21" button id="desktopKCButton">Weź udział w wirtualnym KubeCon North America 24-28.10.2022</a>
```
Comment (mfilocha) [inline content/pl/_index.html:52]:
```suggestion
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna21" button id="desktopKCButton">Weź udział w KubeCon North America 24-28.10.2022</a>
```

Comment (mfilocha) [reply content/pl/_index.html:52]: Hopefully no longer a virtual event...

Comment (mfilocha) [reply content/pl/_index.html:52]: Thanks @Arhell!

**Review by mfilocha — CHANGES_REQUESTED** [review]

File: content/pl/_index.html (hunk 1 above)
Comment (Arhell) [reply content/pl/_index.html]: Done, thanks!

**Review by mfilocha — APPROVED** [review]

**Pull request merged**